fe/fe-core/src/main/java/org/apache/doris/plugin/AuditEvent.java

```diff
@@ -60,6 +60,8 @@ public enum EventType {
     public String ctl = "";
     @AuditField(value = "Db")
     public String db = "";
+    @AuditField(value = "CommandType")
```

Comment:
@eldenmoon  Will it cause sede compatibility problem ?

Reply:
no adding fields will not cause compatibility problem